test/unitTests.js

```diff
@@ -1,10 +1,15 @@
 const RecommendController = require('../travvy-server/src/controllers/RecommendController.js')
+const AuthenticationController = require('../travvy-server/src/controllers/AuthenticationController.js')
```

Comment:
We will need to be a bit careful when merging my PR #60 with this one, there may be some "conficts"

Reply:
I was about to say this. Since Lucy's PR was first, once we merge it you'll have to merge your branch with her changes before we can merge this PR since they deal with the same file.

Reply:
Yes you guys were right! It should be resolved now :)